### ISSUE TITLE
Disk quota on Colossus is exceeded during execution

### ISSUE THREAD

**Issue by oskarvid** [elixir-no-nels/Selma#18]
The workflow is failing on p11 now because the scratch disk quota is exceeded. I have opened a ticket at the help desk, I'm waiting for a reply and will update here once I know more.

**Comment by kjellp** [comment]
Is it a local filesystem that is running full? Or a quota for
/cluster/projects/p11?

How does the error manifest itself/log/error message?


ons. 27. nov. 2019, 08:57 skrev Oskar Vidarsson <notifications@github.com>:

**Comment by oskarvid** [comment]
The scratch disk is on Colossus. See the attached image to see the full error message. You can see that the disk in question is the scratch disk that the job is running on. 
![disk-full](https://user-images.githubusercontent.com/17961690/69851954-4cfebc80-1283-11ea-8d80-b9a98f357c30.png)
TSD support suggested I try to set the TMP_DIR variable to point to the scratch disk so I'm currently running a full test with this change. That shouldn't be the issue since the temp files are already getting written to the scratch disk, you can see this in the screenshot as well. We'll see what the test returns. 

I ran a test with two Coriell split file pairs instead of all seven and that ran 100% with no issues. The workflow runs fine with the NA12878 files, the files that Susanne gave us as well as the unsplit Coriell files so I blame the split ones. My first test on Willow ran fine too.

**Comment by kjellp** [comment]
Thanks for the update, could not see any screenshot though?

**Comment by oskarvid** [comment]
There is a black "line" between the "disk in question is the scratch disk that the job is running on." line and the "TSD support suggested I try to set the TMP_DIR variable to point to the scratch disk so I'm currently running" line, click that.

**Comment by oskarvid** [comment]
I have now concluded the investigation that I have been involved with and here's a brief summary:

Conclusion: There was something wrong with the fastq files that were created on TSD.

Reasoning:
1. I put the Coriell files on Willow and split them up by lane using a oneliner, then I ran the workflow and it was all good.
2. I gave this oneliner to David and he split the same Coriell files on TSD.
3. He tested the workflow, I tested the workflow, and it wouldn't finish.
4. I began suspecting the fastq files and split the Coriell files myself using the same oneliner but the workflow wouldn't finish.
5. I also moved the fastq files that I had created on Willow to TSD and those worked. Therefore I think something odd happened on TSD when we split the original Coriell files by lane.
6. The TMP_DIR suggestion was not the solution because the workflow that worked was executed without setting this variable to a temp directory on the scratch disk and exporting it as suggested by TSD support.
7. This does not explain what was wrong with the split Coriell files from TSD, but at least it works as it should now so I will close this issue.
